.github/workflows/test-ubuntu.yml

```diff
@@ -0,0 +1,44 @@
+name: Run unit tests on ubuntu-latest
```

Comment:
The release action for this repo runs tests on `ubuntu-latest`, `macos-latest`, `windows-latest` so a separate file isn't necessary.